src/script/conversation/ConversationRepository.ts

```diff
@@ -1827,10 +1809,8 @@ export class ConversationRepository {
       return;
     }
     if (!timestamp) {
-      // TODO(federation) find with qualified id
       const conversation = conversationEntity || this.conversationState.findConversation(conversationId);
-      const servertime = this.serverTimeHandler.toServerTimestamp();
```

Comment:
This worked before, because at this point we have not yet sent the message to the server (we do all the checks before actually sending). 
Now the message has started to be sent and already is in the view, this would necessary place the system message before the message being sent